website/settings/local-dist.py

```diff
@@ -51,6 +51,10 @@
         'server': 'http://{}:4203/'.format(EMBER_DOMAIN),
         'path': '/reviews/'
     },
+    'registries': {
```

Comment:
Remove - should be unnecessary given registries has been pulled into ember_osf_web

Reply:
👌